DualityEditor/Forms/NewProjectDialog.cs

```diff
@@ -148,13 +148,16 @@ protected bool CheckInputValid()
 			string targetDir = Path.Combine(this.textBoxFolder.Text, this.textBoxName.Text);
 			if (Directory.Exists(targetDir) || File.Exists(targetDir))
 			{
-				this.textBoxFolder.BackColor = errorColor;
-				MessageBox.Show(
-					GeneralRes.Msg_CreateProjectErrorTargetExists_Desc,
-					GeneralRes.Msg_CreateProjectErrorTargetExists_Caption,
-					MessageBoxButtons.OK,
-					MessageBoxIcon.Warning);
-				return false;
+				if (Directory.EnumerateFiles(targetDir, "*", SearchOption.AllDirectories).Any())

```

Comment:
This will throw an Exception in case the `targetDir` path is actually a file path. You'll need to have a special case for that, so it will just display the error message without doing the directory-files check. Maybe split up the code, so you have one block for existing directories (with your files-check) and one for existing files (without)?